### ISSUE TITLE
How to tweets with image?

### ISSUE THREAD

**Issue by tqnst** [lorensiuswlt/NewAndroidTwitter#1]
Dear loren,

I tweeted succeed status but I can't tweet my message with an image. 
How to do it?

Thank in advanced.

**Comment by DimalPerera** [comment]
Nice work loren,
If you can add image upload part to this, it will really help us.
Thank you

**Comment by tqnst** [comment]
Yes, I added  image upload function to my project with successfully. If you need I can share my code :)

btw, Thank you for excellent library :)
